frontend/src/app/view-claims/view-claims.component.html

```diff
@@ -1,15 +1,68 @@
-        <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
-      </ly-paper>
-    </ly-grid>
-  </ly-grid>
+<br>
+<ly-tabs [alignTabs]="'stretch'">
+
+    <!--STATUS # 0-->
+  <ly-tab>
+    <button ly-tab-label>Awaiting Settlement Option</button>
+    <ly-grid *ngIf="sortClaims(0).length > 0; else noClaims" container [spacing]="16">
+        <ly-grid item [col]="12" *ngFor="let claim of sortClaims(0)">
+          <ly-paper [withClass]="classes.paper" [elevation]="2">
+            <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
+            <p [lyTyp]="'body1'">Vehicle Model: {{ claim.vehicle.model }}</p>
+            <p [lyTyp]="'body1'">Vehicle Color: {{ claim.vehicle.color }}</p>
+            <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
+          </ly-paper>
+        </ly-grid>
+      </ly-grid>
+  </ly-tab>
+
+  <!--STATUS # 1-->
+  <ly-tab>
+    <button ly-tab-label (click)="sortClaims(0)">Finalize Loss</button>
+    <ly-grid *ngIf="sortClaims(1).length > 0; else noClaims" container [spacing]="16">
+        <ly-grid item [col]="12" *ngFor="let claim of sortClaims(1)">
+          <ly-paper [withClass]="classes.paper" [elevation]="2">
+            <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
+            <p [lyTyp]="'body1'">Vehicle Model: {{ claim.vehicle.model }}</p>
+            <p [lyTyp]="'body1'">Vehicle Color: {{ claim.vehicle.color }}</p>
+            <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
+          </ly-paper>
+        </ly-grid>
+      </ly-grid>
+  </ly-tab>
 
+  <!--STATUS # 2-->
+  <ly-tab>
+    <button ly-tab-label (click)="sortClaims(2)">At Auction</button>
+    <ly-grid *ngIf="sortClaims(2).length > 0; else noClaims" container [spacing]="16">
+        <ly-grid item [col]="12" *ngFor="let claim of sortClaims(2)">
+          <ly-paper [withClass]="classes.paper" [elevation]="2">
+            <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
+            <p [lyTyp]="'body1'">Vehicle Model: {{ claim.vehicle.model }}</p>
+            <p [lyTyp]="'body1'">Vehicle Color: {{ claim.vehicle.color }}</p>
+            <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
+          </ly-paper>
+        </ly-grid>
+      </ly-grid>
+  </ly-tab>
+
+  <!--STATUS # 3-->
+  <ly-tab>
+    <button ly-tab-label (click)="sortClaims(3)">Claim Resolved</button>
+    <ly-grid *ngIf="sortClaims(3).length > 0; else noClaims" container [spacing]="16">
+        <ly-grid item [col]="12" *ngFor="let claim of sortClaims(3)">
+          <ly-paper [withClass]="classes.paper" [elevation]="2">
+            <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
+            <p [lyTyp]="'body1'">Vehicle Model: {{ claim.vehicle.model }}</p>
+            <p [lyTyp]="'body1'">Vehicle Color: {{ claim.vehicle.color }}</p>
+            <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
+          </ly-paper>
+        </ly-grid>
+      </ly-grid>
+  </ly-tab>
+</ly-tabs>
+
+<div>
   <ng-template #noClaims>
```

Comment:
Possible change here would be to say "No claims are currently awaiting settlement options" when there are claims, but no claims in that status.